roles/configure/templates/create-stack.sh.j2

```diff
@@ -15,11 +15,16 @@ PARAMETERS=(
 	"ParameterKey=AvailabilityZone,ParameterValue={{ aws_availability_zone }}"
 	"ParameterKey=GatewayInstanceType,ParameterValue={{ gateway_instance_type }}"
 	"ParameterKey=GatewayInstanceCount,ParameterValue={{ gateway_instance_count }}"
+	"ParameterKey=GatewayInstanceRootDiskSize,ParameterValue={{ gateway_instance_root_disk_size_gb }}"
 	"ParameterKey=ControllerInstanceType,ParameterValue={{ controller_instance_type }}"
 	"ParameterKey=ControllerInstanceCount,ParameterValue={{ controller_instance_count }}"
+	"ParameterKey=ControllerInstanceRootDiskSize,ParameterValue={{ controller_instance_root_disk_size_gb }}"
 	"ParameterKey=WorkerInstanceType,ParameterValue={{ worker_instance_type }}"
 	"ParameterKey=WorkerInstanceCount,ParameterValue={{ worker_instance_count }}"
+	"ParameterKey=WorkerInstanceRootDiskSize,ParameterValue={{ worker_instance_root_disk_size_gb }}"
+	"ParameterKey=WorkerInstanceRootDiskIOPs,ParameterValue={{ worker_instance_root_disk_size_gb * 30 }}"
```

Comment:
I don't think it's necessary. It's not easy (or even possible?) to calculate it in CloudFormation, so from the template's perspective it's a required value. I'm not going to establish a new class of vars just yet.